readme.md
<h1 align = center> Aki Password </h1>

<h4 align="center">Simple password manager.</h4>

<p align="center">

 <a href="https://www.python.org/downloads/">
    <img alt="PyPI - Python Version" src="https://img.shields.io/pypi/pyversions/Red-Discordbot">
  </a>
  <a href="https://opensource.org/licenses/MIT">
     <img src="https://img.shields.io/badge/License-MIT-yellow.svg" alt="MIT License">
  </a>
</p>

<p align="center">
  <a href="#overview">Overview</a>
  •
  <a href="#installation">Installation</a>
  •
  <a href="#discord">Discord</a>
  •
  <a href="#license">License</a>
</p>

# Overview

Aki Password is a simple password manager where you can safely store your data.
![Passwords menu](https://github.com/ilyhalight/Aki-Password/blob/master/screenshots/screenshot_main.png) 

**Now done:**
 - Registration and authorization system
 - Notification system
 - User interface
 - Data display from db
 - Password adding system
 
**Not implemented:**
 - Password hashing system 

# Installation

**The following operating systems are officially supported:**
 - [Windows](https://vk.com/topic-201228277_47491764)

 # Discord

Hopefully this project will be fully completed by May 2021.

In our discord you can find information and help: https://discord.gg/Vh3VcaEv23

# License

Released under the [MIT License](https://opensource.org/licenses/MIT).


changelog.md
# Alpha 0.1 (2021-03-27)
- Added aki_accounts.sql database
- Added .gitignore
- Made the application base
- Added an authorization system
- Added registration system
- Added a notification system
- Added user interface
- Added a template for the system for adding passwords
# Alpha 0.2 (2021-03-29)
- Added requirements.txt
- Added readme.md
- Added changelog.md
- Added a system for adding passwords (does not work correctly)
- Added display of data from the database
- Minor edits in ui_main.py
- Changed database columns
# Alpha 0.3 (2021-03-30)
- Added password adding system (Fully working)
- Added commands for more convenient work with the database
# Alpha 0.3.1 (2021-03-30)
- Changed default values for "sitename, sitelogin, sitepassword"
- Added autoreplacement of default values with user-created ones
- Fixed a bug with the program crash when the number of rows in the user database was more than 12
# Alpha 0.4 (2021-04-22)
- Added removal of passwords from the program
- Some other minor changes

ui/ui_splashscreen.py
# -*- coding: utf-8 -*-

# Form implementation generated from reading ui file 'ui_splashscreen.ui'
#
# Created by: PyQt5 UI code generator 5.15.2
#
# WARNING: Any manual changes made to this file will be lost when pyuic5 is
# run again.  Do not edit this file unless you know what you are doing.


from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Aki_splashscreen(object):
    def setupUi(self, Aki_splashscreen):
        Aki_splashscreen.setObjectName("Aki_splashscreen")
        Aki_splashscreen.resize(684, 401)
        self.centralwidget = QtWidgets.QWidget(Aki_splashscreen)
        self.centralwidget.setObjectName("centralwidget")
        self.ss_background = QtWidgets.QFrame(self.centralwidget)
        self.ss_background.setGeometry(QtCore.QRect(10, 10, 660, 380))
        self.ss_background.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        self.ss_background.setStyleSheet("QFrame {\n"
"    background-color:rgb(24, 24, 24);\n"
"    color:rgb(177, 177, 177);\n"
"    border-radius:  20px;\n"
"    border: None\n"
"}")
        self.ss_background.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.ss_background.setFrameShadow(QtWidgets.QFrame.Raised)
        self.ss_background.setObjectName("ss_background")
        self.ss_label_title = QtWidgets.QLabel(self.ss_background)
        self.ss_label_title.setGeometry(QtCore.QRect(0, 60, 651, 51))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(40)
        self.ss_label_title.setFont(font)
        self.ss_label_title.setStyleSheet("color: rgb(230, 52, 98);")
        self.ss_label_title.setAlignment(QtCore.Qt.AlignCenter)
        self.ss_label_title.setObjectName("ss_label_title")
        self.ss_label_description = QtWidgets.QLabel(self.ss_background)
        self.ss_label_description.setGeometry(QtCore.QRect(7, 120, 651, 31))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(18)
        self.ss_label_description.setFont(font)
        self.ss_label_description.setAlignment(QtCore.Qt.AlignCenter)
        self.ss_label_description.setObjectName("ss_label_description")
        self.ss_progressBar = QtWidgets.QProgressBar(self.ss_background)
        self.ss_progressBar.setGeometry(QtCore.QRect(50, 280, 561, 23))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(8)
        self.ss_progressBar.setFont(font)
        self.ss_progressBar.setStyleSheet("QProgressBar {\n"
"    background-color: #0e0e0e;\n"
"    color: rgb(177, 177, 177);\n"
"    border-style: none;\n"
"    border-radius: 10px;\n"
"    text-align: center\n"
"}\n"
"QProgressBar::chunk{\n"
"    border-radius: 10px;\n"
"    background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1,y2:0.523, stop:0 rgba(245, 112, 147, 255), stop:1 rgba(230, 52, 98, 255));\n"
"}")
        self.ss_progressBar.setProperty("value", 24)
        self.ss_progressBar.setObjectName("ss_progressBar")
        self.label_loading = QtWidgets.QLabel(self.ss_background)
        self.label_loading.setGeometry(QtCore.QRect(-1, 310, 661, 21))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(12)
        self.label_loading.setFont(font)
        self.label_loading.setStyleSheet("")
        self.label_loading.setAlignment(QtCore.Qt.AlignCenter)
        self.label_loading.setObjectName("label_loading")
        self.ss_copyright = QtWidgets.QLabel(self.ss_background)
        self.ss_copyright.setGeometry(QtCore.QRect(54, 349, 561, 21))
        font = QtGui.QFont()
        font.setFamily("Segoe UI")
        self.ss_copyright.setFont(font)
        self.ss_copyright.setStyleSheet("")
        self.ss_copyright.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.ss_copyright.setObjectName("ss_copyright")
        Aki_splashscreen.setCentralWidget(self.centralwidget)

        self.retranslateUi(Aki_splashscreen)
        QtCore.QMetaObject.connectSlotsByName(Aki_splashscreen)

    def retranslateUi(self, Aki_splashscreen):
        _translate = QtCore.QCoreApplication.translate
        Aki_splashscreen.setWindowTitle(_translate("Aki_splashscreen", "MainWindow"))
        self.ss_label_title.setText(_translate("Aki_splashscreen", "<strong>AKI</strong> REWORKED"))
        self.ss_label_description.setText(_translate("Aki_splashscreen", "<html><head/><body><p><span style=\" font-weight:600;\">ADMIN PANEL</span></p></body></html>"))
        self.label_loading.setText(_translate("Aki_splashscreen", "loading..."))
        self.ss_copyright.setText(_translate("Aki_splashscreen", "Aki © 2021 Все права защищены"))


ui/ui_main1.py
# -*- coding: utf-8 -*-

# Form implementation generated from reading ui file 'ui_main1.ui'
#
# Created by: PyQt5 UI code generator 5.15.2
#
# WARNING: Any manual changes made to this file will be lost when pyuic5 is
# run again.  Do not edit this file unless you know what you are doing.


from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Aki_main(object):
    def setupUi(self, Aki_main):
        Aki_main.setObjectName("Aki_main")
        Aki_main.resize(961, 578)
        Aki_main.setStyleSheet("QMainWindow {\n"
"    border: none;\n"
"    border-radius: 25px\n"
"}")
        self.centralwidget = QtWidgets.QWidget(Aki_main)
        self.centralwidget.setObjectName("centralwidget")
        self.background_main = QtWidgets.QFrame(self.centralwidget)
        self.background_main.setGeometry(QtCore.QRect(10, 10, 941, 561))
        self.background_main.setStyleSheet("QFrame {\n"
"    background-color:rgb(24, 24, 24);\n"
"    color:rgb(177, 177, 177);\n"
"    border-radius:  20px\n"
"}")
        self.background_main.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.background_main.setFrameShadow(QtWidgets.QFrame.Raised)
        self.background_main.setObjectName("background_main")
        self.poloska = QtWidgets.QFrame(self.background_main)
        self.poloska.setGeometry(QtCore.QRect(0, 50, 941, 1))
        self.poloska.setStyleSheet("QFrame {\n"
"    background-color:rgb(33, 33, 33);\n"
"}")
        self.poloska.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.poloska.setFrameShadow(QtWidgets.QFrame.Plain)
        self.poloska.setObjectName("poloska")
        self.btn_close = QtWidgets.QPushButton(self.background_main)
        self.btn_close.setGeometry(QtCore.QRect(910, 15, 16, 16))
        self.btn_close.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.btn_close.setStyleSheet("QPushButton {\n"
"    background: transparent\n"
"}")
        self.btn_close.setText("")
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("icons/aki_close.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btn_close.setIcon(icon)
        self.btn_close.setIconSize(QtCore.QSize(13, 13))
        self.btn_close.setObjectName("btn_close")
        self.img_logo = QtWidgets.QPushButton(self.background_main)
        self.img_logo.setGeometry(QtCore.QRect(20, 15, 24, 24))
        self.img_logo.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        self.img_logo.setStyleSheet("QPushButton {\n"
"    background: transparent\n"
"}")
        self.img_logo.setText("")
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("icons/aki_logo.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.img_logo.setIcon(icon1)
        self.img_logo.setIconSize(QtCore.QSize(18, 18))
        self.img_logo.setObjectName("img_logo")
        self.logo_text = QtWidgets.QLabel(self.background_main)
        self.logo_text.setGeometry(QtCore.QRect(45, 20, 171, 16))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.logo_text.setFont(font)
        self.logo_text.setStyleSheet("QLabel {\n"
"    color: #fff\n"
"}")
        self.logo_text.setObjectName("logo_text")
        self.main_window_copyright = QtWidgets.QLabel(self.background_main)
        self.main_window_copyright.setGeometry(QtCore.QRect(40, 530, 851, 21))
        font = QtGui.QFont()
        font.setFamily("Segoe UI")
        self.main_window_copyright.setFont(font)
        self.main_window_copyright.setStyleSheet("")
        self.main_window_copyright.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.main_window_copyright.setObjectName("main_window_copyright")
        self.label = QtWidgets.QLabel(self.background_main)
        self.label.setGeometry(QtCore.QRect(46, 73, 881, 31))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(16)
        self.label.setFont(font)
        self.label.setAlignment(QtCore.Qt.AlignCenter)
        self.label.setObjectName("label")
        self.frame_3 = QtWidgets.QFrame(self.background_main)
        self.frame_3.setGeometry(QtCore.QRect(170, 110, 651, 423))
        self.frame_3.setMinimumSize(QtCore.QSize(0, 150))
        self.frame_3.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_3.setObjectName("frame_3")
        self.tableWidget = QtWidgets.QTableWidget(self.frame_3)
        self.tableWidget.setGeometry(QtCore.QRect(0, -10, 651, 423))
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.tableWidget.sizePolicy().hasHeightForWidth())
        self.tableWidget.setSizePolicy(sizePolicy)
        palette = QtGui.QPalette()
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Text, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.ButtonText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.NoBrush)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Base, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177, 128))
        brush.setStyle(QtCore.Qt.NoBrush)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.PlaceholderText, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Text, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.ButtonText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.NoBrush)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Base, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177, 128))
        brush.setStyle(QtCore.Qt.NoBrush)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.PlaceholderText, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Text, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.ButtonText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.NoBrush)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Base, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(177, 177, 177, 128))
        brush.setStyle(QtCore.Qt.NoBrush)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.PlaceholderText, brush)
        self.tableWidget.setPalette(palette)
        font = QtGui.QFont()
        font.setFamily("Acrom")
        self.tableWidget.setFont(font)
        self.tableWidget.setStyleSheet("QTableWidget {    \n"
"    background-color: transparent;\n"
"    padding: 10px;\n"
"    border-radius: 5px;\n"
"    gridline-color: transparent;\n"
"}\n"
"QTableWidget::item{\n"
"    border-color: transparent;\n"
"    padding-left: 5px;\n"
"    padding-right: 5px;\n"
"    gridline-color: transparent;\n"
"}\n"
"QTableWidget::item:selected{\n"
"    background-color: transparent;\n"
"}\n"
"QScrollBar:horizontal {\n"
"    border: none;\n"
"    background: transparent;\n"
"    height: 14px;\n"
"    margin: 0px 21px 0 21px;\n"
"    border-radius: 0px;\n"
"}\n"
" QScrollBar:vertical {\n"
"    border: none;\n"
"    background: #fff;\n"
"    width: 14px;\n"
"    margin: 21px 0 21px 0;\n"
"    border-radius: 0px;\n"
" }\n"
"QHeaderView::section{\n"
"    Background-color: transparent;\n"
"    max-width: 30px;\n"
"    border: 1px solid transparent;\n"
"    border-style: none;\n"
"    border-bottom: 1px solid transparent;\n"
"    border-right: 1px solid transparent;\n"
"}\n"
"QTableWidget::horizontalHeader {    \n"
"    background-color: transparent;\n"
"}\n"
"QHeaderView::section:horizontal\n"
"{\n"
"    border: 1px solid transparent;\n"
"    background-color: transparent;\n"
"    padding: 3px;\n"
"    border-top-left-radius: 7px;\n"
"    border-top-right-radius: 7px;\n"
"}\n"
"QHeaderView::section:vertical\n"
"{\n"
"    border: 1px solid transparent;\n"
"}\n"
"")
        self.tableWidget.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.tableWidget.setVerticalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOn)
        self.tableWidget.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
        self.tableWidget.setSizeAdjustPolicy(QtWidgets.QAbstractScrollArea.AdjustToContents)
        self.tableWidget.setEditTriggers(QtWidgets.QAbstractItemView.NoEditTriggers)
        self.tableWidget.setAlternatingRowColors(False)
        self.tableWidget.setSelectionMode(QtWidgets.QAbstractItemView.SingleSelection)
        self.tableWidget.setSelectionBehavior(QtWidgets.QAbstractItemView.SelectRows)
        self.tableWidget.setShowGrid(True)
        self.tableWidget.setGridStyle(QtCore.Qt.SolidLine)
        self.tableWidget.setObjectName("tableWidget")
        self.tableWidget.setColumnCount(3)
        self.tableWidget.setRowCount(16)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Acrom")
        item.setFont(font)
        self.tableWidget.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Acrom")
        item.setFont(font)
        self.tableWidget.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Acrom")
        item.setFont(font)
        self.tableWidget.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Acrom")
        item.setFont(font)
        self.tableWidget.setHorizontalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Acrom")
        item.setFont(font)
        self.tableWidget.setItem(0, 0, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Acrom")
        item.setFont(font)
        self.tableWidget.setItem(0, 1, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Acrom")
        item.setFont(font)
        self.tableWidget.setItem(0, 2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setItem(1, 0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setItem(2, 0, item)
        self.tableWidget.horizontalHeader().setVisible(True)
        self.tableWidget.horizontalHeader().setCascadingSectionResizes(True)
        self.tableWidget.horizontalHeader().setDefaultSectionSize(200)
        self.tableWidget.horizontalHeader().setSortIndicatorShown(False)
        self.tableWidget.horizontalHeader().setStretchLastSection(True)
        self.tableWidget.verticalHeader().setVisible(False)
        self.tableWidget.verticalHeader().setCascadingSectionResizes(False)
        self.tableWidget.verticalHeader().setHighlightSections(False)
        self.tableWidget.verticalHeader().setStretchLastSection(True)
        self.btn_copy_pass = QtWidgets.QPushButton(self.frame_3)
        self.btn_copy_pass.setGeometry(QtCore.QRect(560, 30, 21, 21))
        self.btn_copy_pass.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.btn_copy_pass.setStyleSheet("QPushButton {\n"
"    background: transparent\n"
"}")
        self.btn_copy_pass.setText("")
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap("icons/aki_copy.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btn_copy_pass.setIcon(icon2)
        self.btn_copy_pass.setObjectName("btn_copy_pass")
        self.btn_delete_user = QtWidgets.QPushButton(self.frame_3)
        self.btn_delete_user.setGeometry(QtCore.QRect(590, 30, 21, 21))
        self.btn_delete_user.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.btn_delete_user.setToolTipDuration(-1)
        self.btn_delete_user.setStyleSheet("QPushButton {\n"
"    background: transparent\n"
"}")
        self.btn_delete_user.setText("")
        icon3 = QtGui.QIcon()
        icon3.addPixmap(QtGui.QPixmap("icons/aki_delete.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btn_delete_user.setIcon(icon3)
        self.btn_delete_user.setObjectName("btn_delete_user")
        self.btn_add_user = QtWidgets.QPushButton(self.frame_3)
        self.btn_add_user.setGeometry(QtCore.QRect(530, 360, 56, 56))
        self.btn_add_user.setMinimumSize(QtCore.QSize(56, 56))
        self.btn_add_user.setMaximumSize(QtCore.QSize(56, 56))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(56)
        self.btn_add_user.setFont(font)
        self.btn_add_user.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.btn_add_user.setStyleSheet("QPushButton {\n"
"    border-radius: 26px;\n"
"    background-color: rgb(44, 49, 60);\n"
"    border: 5px solid rgb(39, 44, 54);\n"
"    background-position: center;\n"
"    background-repeat: no-repeat;\n"
"    color: #3ca24f\n"
"}")
        self.btn_add_user.setObjectName("btn_add_user")
        self.frame_remove = QtWidgets.QFrame(self.background_main)
        self.frame_remove.setGeometry(QtCore.QRect(80, 60, 741, 471))
        self.frame_remove.setStyleSheet("QFrame {\n"
"    background-color:rgba(24, 24, 24, 0.7);\n"
"    color:rgb(177, 177, 177);\n"
"    border-radius:  20px\n"
"}")
        self.frame_remove.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_remove.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_remove.setObjectName("frame_remove")
        self.l_remove_row = QtWidgets.QLabel(self.frame_remove)
        self.l_remove_row.setGeometry(QtCore.QRect(0, 10, 741, 41))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(30)
        self.l_remove_row.setFont(font)
        self.l_remove_row.setAlignment(QtCore.Qt.AlignCenter)
        self.l_remove_row.setObjectName("l_remove_row")
        self.remove_row = QtWidgets.QLineEdit(self.frame_remove)
        self.remove_row.setGeometry(QtCore.QRect(230, 180, 261, 41))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        self.remove_row.setFont(font)
        self.remove_row.setStyleSheet("QLineEdit {\n"
"    border: none;\n"
"    background-color: #0e0e0e;\n"
"    border-radius: 7px;\n"
"    color: #fff\n"
"}")
        self.remove_row.setInputMask("")
        self.remove_row.setObjectName("remove_row")
        self.l_number_row = QtWidgets.QLabel(self.frame_remove)
        self.l_number_row.setGeometry(QtCore.QRect(230, 160, 251, 16))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        font.setPointSize(12)
        self.l_number_row.setFont(font)
        self.l_number_row.setObjectName("l_number_row")
        self.remove_complete = QtWidgets.QPushButton(self.frame_remove)
        self.remove_complete.setGeometry(QtCore.QRect(230, 360, 111, 31))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        self.remove_complete.setFont(font)
        self.remove_complete.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.remove_complete.setStyleSheet("QPushButton {\n"
"    color: #fff;\n"
"    border: none;\n"
"    border-radius: 10.5px;\n"
"    background-color: rgb(230, 52, 98)\n"
"}")
        self.remove_complete.setObjectName("remove_complete")
        self.remove_cancel = QtWidgets.QPushButton(self.frame_remove)
        self.remove_cancel.setGeometry(QtCore.QRect(380, 360, 111, 31))
        font = QtGui.QFont()
        font.setFamily("Acrom")
        self.remove_cancel.setFont(font)
        self.remove_cancel.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.remove_cancel.setStyleSheet("QPushButton {\n"
"    color: #fff;\n"
"    border: none;\n"
"    border-radius: 10.5px;\n"
"    background-color: rgb(108, 109,112);\n"
"}")
        self.remove_cancel.setObjectName("remove_cancel")
        Aki_main.setCentralWidget(self.centralwidget)

        self.retranslateUi(Aki_main)
        QtCore.QMetaObject.connectSlotsByName(Aki_main)

    def retranslateUi(self, Aki_main):
        _translate = QtCore.QCoreApplication.translate
        Aki_main.setWindowTitle(_translate("Aki_main", "MainWindow"))
        self.logo_text.setText(_translate("Aki_main", "Aki"))
        self.main_window_copyright.setText(_translate("Aki_main", "Aki © 2021 Все права защищены"))
        self.label.setText(_translate("Aki_main", "СОХРАНЕННЫЕ ПАРОЛИ"))
        self.tableWidget.setSortingEnabled(False)
        item = self.tableWidget.verticalHeaderItem(0)
        item.setText(_translate("Aki_main", "New Row"))
        item = self.tableWidget.verticalHeaderItem(1)
        item.setText(_translate("Aki_main", "New Row"))
        item = self.tableWidget.horizontalHeaderItem(0)
        item.setText(_translate("Aki_main", "Сайт"))
        item = self.tableWidget.horizontalHeaderItem(1)
        item.setText(_translate("Aki_main", "Имя пользователя"))
        item = self.tableWidget.horizontalHeaderItem(2)
        item.setText(_translate("Aki_main", "Пароль"))
        __sortingEnabled = self.tableWidget.isSortingEnabled()
        self.tableWidget.setSortingEnabled(False)
        item = self.tableWidget.item(0, 0)
        item.setText(_translate("Aki_main", "Site"))
        item = self.tableWidget.item(0, 1)
        item.setText(_translate("Aki_main", "User"))
        item = self.tableWidget.item(0, 2)
        item.setText(_translate("Aki_main", "Pass"))
        self.tableWidget.setSortingEnabled(__sortingEnabled)
        self.btn_add_user.setText(_translate("Aki_main", "+"))
        self.l_remove_row.setText(_translate("Aki_main", "Удалить данные"))
        self.remove_row.setPlaceholderText(_translate("Aki_main", "От 0 до бесконечности"))
        self.l_number_row.setText(_translate("Aki_main", "Номер строки"))
        self.remove_complete.setText(_translate("Aki_main", "Сохранить"))
        self.remove_cancel.setText(_translate("Aki_main", "Отмена"))
